modules/reference/pages/default-environment-variables.adoc

```diff
@@ -41,6 +41,11 @@ The following table lists the default Open Liberty environment variables and giv
   This log file is only used if the xref:reference:command/server-start.adoc[server start] command is run in the
   background through the start action.
 
+|VARIABLE_SOURCE_DIRS
+
+| The directories that contain files to be loaded as configuration variables.
+  The default value is `${WLP_CONFIG_DIR}/variables`.
```

Comment:
in the server-configuration-overview.adoc, the default value is `${server.config.dir}/variables`
We dont have an env variable `${WLP_CONFIG_DIR}` documented here. 

It might be the case that these are the same default location, `${server.config.dir}`=`${WLP_CONFIG_DIR}`
we need to clarify that, and if `WLP_CONFIG_DIR` is a default environment variable, it should be added to this table. You can merge this PR before that is sorted but need to clarify w/ SME as part of the review process for the doc